src/main/java/ca/corefacility/bioinformatics/irida/service/IridaClientDetailsService.java

```diff
@@ -16,7 +16,7 @@
 	/**
 	 * {@inheritDoc}
 	 */
-	public ClientDetails loadClientByClientId(String clientId) throws ClientRegistrationException;
```

Comment:
Why was "public" removed from the method signatures?

Reply:
Because Intellij on my Mac does crazy things.

Reply:
Fixed